jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/tables/SimpleTable.java

```diff
@@ -4,14 +4,15 @@
 import com.epam.jdi.light.elements.complex.WebList;
 import com.epam.jdi.light.elements.complex.table.BaseTable;
 import com.epam.jdi.light.vuetify.asserts.tables.SimpleTableAssert;
+import com.epam.jdi.light.vuetify.interfaces.HasTheme;
 import com.epam.jdi.light.vuetify.interfaces.IsDense;
 
 /**
  * To see an example of Simple Table web element please visit https://vuetifyjs.com/en/components/simple-tables/
  **/
 
 public class SimpleTable<T extends SimpleTable<T, A>, A extends SimpleTableAssert<T, A>>
-    extends BaseTable<T, A> implements IsDense {
+    extends BaseTable<T, A> implements IsDense, HasTheme {
 
     @Override
     public A is() {
```

Comment:
Could you please clarify these points mentioned in our treceability matrix:
No data text / No results text / Loading + Loading text
Group by

Reply:
Tasks for adding these features on the test site https://github.com/jdi-testing/jdi-light/issues/4705 and in vuetify tests https://github.com/jdi-testing/jdi-light/issues/4706 were created. Thanks!